### PR TITLE
Add 3D-GAN PyTorch implementation.

### PR DESCRIPTION
**Pull request by SomTambe**
- [x] README.md
- [x]  Different files according to format, with command-line arguments as well.

[Som Tambe](https://github.com/SomTambe/)

### CODE REVIEW

File: generative_models/3D-GAN_PyTorch/README.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+# **PyTorch implementation of 3D-GAN**
+## Usage
+```bash
+> python main.py --arguments
+```
```
Comment (nirmal-suthar) [inline generative_models/3D-GAN_PyTorch/README.md]: add script to download 'modelnet10.npy.gz' something like `wget <download link>`

File: generative_models/3D-GAN_PyTorch/README.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
+# **PyTorch implementation of 3D-GAN**
+## Usage
+```bash
+> python main.py --arguments
```
Comment (nirmal-suthar) [inline generative_models/3D-GAN_PyTorch/README.md]: you may add output of `python main.py --help` here, to indicate available arguments

**Comment by nirmal-suthar** [comment]
- Remove .DS_Store file in your commit
- And add your model in the folder named 3D_vision
- Rename your model folder to 3DGAN_PyTorch

File: generative_models/3D-GAN_PyTorch/README.md (hunk 2 above)
Comment (SomTambe) [reply generative_models/3D-GAN_PyTorch/README.md]: Yes I will do that

File: generative_models/3D-GAN_PyTorch/README.md (hunk 1 above)
Comment (SomTambe) [reply generative_models/3D-GAN_PyTorch/README.md]: Yes I will do that, but that does not work. I mean the downloaded file shows some error when we try to unzip using gzip on python. 

If you want, I will still add the link ?

**Comment by SomTambe** [comment]
I have done the said changes, please review them and let me know of any problem :)

**Comment by SomTambe** [comment]
Done final changes. Please review.

**Pull request merged**